liquibase-core/src/main/java/liquibase/configuration/core/InteractivePromptingValueProvider.java

```diff
@@ -0,0 +1,30 @@
+package liquibase.configuration.core;
+
+import liquibase.configuration.AbstractMapConfigurationValueProvider;
+
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+/**
+ * This value provider handles values obtained during interactive CLI prompting.
+ */
+public class InteractivePromptingValueProvider extends AbstractMapConfigurationValueProvider {
```

Comment:
This doesn't seem to actually do any interactive prompting?

If it's really just a holder for actual values, is it better to just be putting the key/values into Scope vs. adding this class?

Reply:
This source file was moved to Pro, but apparently this branch still had the original version.  I am deleting it.